src/languages/en.js

```diff
@@ -761,4 +761,14 @@ export default {
     emojiPicker: {
         skinTonePickerLabel: 'Change default skin tone',
     },
+    keyboardShortCutModal: {
+        title: 'Keyboard Shortcuts',
+        subtitle: 'Do you expense reports faster with these handy keyboard shortcuts',
```

Comment:
```suggestion
        subtitle: 'Do your expense reports faster with these handy keyboard shortcuts',
```

Reply:
@Expensify/marketing Can you please review the copy in this keyboard shortcuts modal? I'm concerned "Do your expense reports faster..." doesn't capture our myriad use cases.

Reply:
```suggestion
        subtitle: 'Save time with these handy keyboard shortcuts:',
```